api/src/main/java/org/openmrs/module/appointmentscheduling/api/AppointmentService.java

```diff
@@ -916,10 +916,10 @@ Integer getHistoryCountByConditions(Date fromDate, Date endDate,
 	List<AppointmentType> getAllAppointmentTypesSorted(boolean includeRetired);
 
 	/**
-	 * Retrieves list of unvoided appointments that their current status is one of the given states.
+	 * Retrieves list of unvoided(restored) appointments that their current status is one of the given states.
```

Comment:
Instead of "(restored)", should be "(not deleted)", see my comment on TRUNK-2299